libmozdata/patchanalysis.py

```diff
@@ -696,3 +701,74 @@ def handler_revision(json, data):
         del info[r]
 
     return info
+
+
+def parse_uplift_comment(text, bugzilla_url, bug_id=None):
```

Comment:
Unless you need it, we can remove the `bugzilla_url` parameter and always assume it is Bugzilla.URL.